### PR TITLE
docs: add Slack to the communication channels

### PR DESCRIPTION
**Pull request by jbampton**
### What this PR does / why we need it:

Adds all the communication channels to the README's in the table.

### Pre-submission checklist:

* [X] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -271,7 +271,7 @@ CentOS 7, Ubuntu 16.04, Ubuntu 18.04, Debian 9, Debian 10, macOS, **ARM64** Ubun
 | :------------------------------------ | :-------------------------------------- | :--------------------- |
 | 项目归属                              | Apache 软件基金会                       | Kong Inc.              |
 | 技术架构                              | Nginx + etcd                            | Nginx + postgres       |
-| 交流渠道                              | 微信群、QQ 群、邮件列表、GitHub、meetup | GitHub、论坛、freenode |
+| 交流渠道                              | 微信群、QQ 群、邮件列表、GitHub、Slack、meetup | GitHub、论坛、freenode |
```
Comment (juzhiyuan) [inline README_CN.md:274]: 🤔 Maybe we could add the related links to GitHub Issue/Slack?

**Review by juzhiyuan — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -269,27 +269,27 @@ Using AWS's 8 core server, APISIX's QPS reach to 140,000 with a latency of only
...
-| Speed limit for a specified time window                         | Yes                                               | No                      |
-| Support any Nginx variable as routing condition                 | Yes                                               | No                      |
+| **Features**                                                    | **Apache APISIX**                                        | **Kong**                |
+| :-------------------------------------------------------------- | :------------------------------------------------------- | :---------------------- |
+| Belongs to                                                      | Apache Software Foundation                               | Kong Inc.               |
+| Tech Architecture                                               | Nginx + etcd                                             | Nginx + postgres        |
+| Communication channels                                          | Mail list, Wechat group, QQ group, GitHub, Slack, meetup | GitHub, freenode, forum |
```
Comment (Yiyiyimu) [inline README.md:276]: Hi it might be a good idea to align the last vertical bar 🤣  I guess some IDE plugin could do that

**Comment by membphis** [comment]
need to merge the master branch

**Comment by moonming** [comment]
@jbampton please rebase the master branch, thx

File: docs/zh/latest/README.md [hunk 3]
```diff
@@ -271,27 +271,27 @@ CentOS 7, Ubuntu 16.04, Ubuntu 18.04, Debian 9, Debian 10, macOS, **ARM64** Ubun
...
-| 单核 QPS (开启限流和 prometheus 插件) | 18000                                   | 1700                   |
-| 平均延迟                              | 0.2 毫秒                                | 2 毫秒                 |
-| 支持 Dubbo 代理                       | 是                                      | 否                     |
-| 配置回滚                              | 是                                      | 否                     |
-| 支持生命周期的路由                    | 是                                      | 否                     |
-| 插件热更新                            | 是                                      | 否                     |
-| 用户自定义：负载均衡算法、路由        | 是                                      | 否                     |
+| 项目归属                               | Apache 软件基金会                         | Kong Inc.              |
+| 技术架构                               | Nginx + etcd                            | Nginx + postgres       |
+| 交流渠道                               | 微信群、QQ 群、邮件列表、[GitHub](https://github.com/apache/apisix/issues)、[Slack](https://join.slack.com/t/the-asf/shared_invite/zt-mrougyeu-2aG7BnFaV0VnAT9_JIUVaA)、meetup | GitHub、论坛、freenode |
```
Comment (juzhiyuan) [inline docs/zh/latest/README.md:278]: Hi, I just generated a new link: https://join.slack.com/t/the-asf/shared_invite/zt-nggtva4i-hDCsW1S35MuZ2g_2DgVDGg

Slack invitation link may get expired :)

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Review by tokers — APPROVED** [review]

**Pull request merged**